backend/btrixcloud/pages.py

```diff
@@ -577,7 +577,13 @@ async def get_qa_run_aggregate_counts(
             boundaries.append(1.1)
 
         aggregate = [
-            {"$match": {"crawl_id": crawl_id}},
+            {
+                "$match": {
+                    "crawl_id": crawl_id,
+                    "isFile": {"$ne": True},
+                    "isError": {"$ne": True},
```

Comment:
A question is if we want to return the `No data` bucket at all, since it is now being ignored on the frontend (considered 'incomplete' / unfinished QA run).

If so, then we could change this to:
```suggestion
                    f"qa.{qa_run_id}.{key}": {"$ne": None}
```
and remove the isFile / isError checks, which would then filter out any pages with no data for that dimension and have 'No data' bucket

Reply:
For completeness, may be nice to keep the 'No data' it in the API

Reply:
Yeah the "No data" is a bit of a fallback in the way the stats are being generated, so good to keep it there, but makes sense to filter these out from what's going in the meter to begin with!

Reply:
Maybe worth keeping the separate isError/isFile checks for the moment, just to see if there are any other "No Data" pages that are slipping through still?

Reply:
ok, so lets leave this as is!